### PR TITLE
Sauce IPSec Proxy edits

### PR DESCRIPTION
**Pull request by kimsauce**
### Description

Edits for clarity

### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Documentation fix (typos, incorrect content, missing content, etc.)

### Checklist

- [ ] I have read the [contributing](https://github.com/saucelabs/sauce-docs/blob/main/docs/contributing.md) document.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: docs/secure-connections/ipsec-vpn.md [hunk 1]
```diff
@@ -113,24 +113,34 @@ WebSocket servers with self-signed certificates are not supported.
...
+  caps.setCapability("tunnelIdentifier", "$TUNNEL_IDENTIFIER");
+  caps.setCapability("parentTunnel","$SAUCE_USERNAME");
+  ```
+
+#### Espresso and XCUITest Frameworks
+If you're working in Espresso or XCUITest, you'll need to use [`saucectl`](/dev/cli/saucectl/) to run all tests on Sauce Labs.
+
+1. First, you'll need to specify the [data center](/basics/data-center-endpoints) location of the device you're testing on. There are two ways to do this:
+   * Specify the `region` in your YAML configuration file (see [Espresso config > `region`](/mobile-apps/automated-testing/espresso-xcuitest/espresso/#region) or [XCUITest config > `region`](/mobile-apps/automated-testing/espresso-xcuitest/xcuitest/#region)); or
+   * Use the [`--region` flag](/dev/cli/saucectl/run/#--region) in conjunction with the `saucectl` run command at test runtime.
```
Comment (lysannep) [inline docs/secure-connections/ipsec-vpn.md:134]:
```suggestion
   * Use the [`--region`](/dev/cli/saucectl/run/#--region) flag in conjunction with the `saucectl` run command at test runtime.
```

File: docs/secure-connections/ipsec-vpn.md [hunk 2]
```diff
@@ -113,24 +113,34 @@ WebSocket servers with self-signed certificates are not supported.
...
+  ```java title="Java example"
+  MutableCapabilities caps = new MutableCapabilities();
+  caps.setCapability("tunnelIdentifier", "$TUNNEL_IDENTIFIER");
+  caps.setCapability("parentTunnel","$SAUCE_USERNAME");
+  ```
+
+#### Espresso and XCUITest Frameworks
+If you're working in Espresso or XCUITest, you'll need to use [`saucectl`](/dev/cli/saucectl/) to run all tests on Sauce Labs.
+
+1. First, you'll need to specify the [data center](/basics/data-center-endpoints) location of the device you're testing on. There are two ways to do this:
```
Comment (lysannep) [inline docs/secure-connections/ipsec-vpn.md:132]:
```suggestion
1. Specify the [data center](/basics/data-center-endpoints) location of the device you're testing on. There are two ways to do this:
```

File: docs/secure-connections/ipsec-vpn.md [hunk 3]
```diff
@@ -113,24 +113,34 @@ WebSocket servers with self-signed certificates are not supported.
...
+   * Specify the `region` in your YAML configuration file (see [Espresso config > `region`](/mobile-apps/automated-testing/espresso-xcuitest/espresso/#region) or [XCUITest config > `region`](/mobile-apps/automated-testing/espresso-xcuitest/xcuitest/#region)); or
+   * Use the [`--region` flag](/dev/cli/saucectl/run/#--region) in conjunction with the `saucectl` run command at test runtime.
+      ```bash
+      saucectl run --region eu-central-1
+      ```
+2. Next, you'll need to assign your Sauce IPSec Proxy tunnel to your job.  There are two ways to do this:
```
Comment (lysannep) [inline docs/secure-connections/ipsec-vpn.md:138]:
```suggestion
2. Assign your Sauce IPSec Proxy tunnel to your job.  There are two ways to do this:
```

**Review by lysannep — APPROVED** [review]
Looks good to me! I added a few suggestions.

**Comment by kimsauce** [comment]
> Looks good to me! I added a few suggestions.

@lysannep Thank you so much! These are great catches.

**Pull request merged**